### DISCUSSION TITLE
packageFileDir is empty string

### DISCUSSION THREAD

**Discussion by songkeys** [renovatebot/renovate#9188] [Request Help]
- version: 24.89.1
- `packageFileDir` is empty string.

my commands:

```
cd {{packageFileDir}} && ...
```

the output log shows:

```
cd  && ...
```

**Comment by rarkins** [comment]
Is the package file in the root directory?

**Reply by songkeys** [reply]
is the root directory supposed to be an empty string? Can we make this an absolute path to be clearer?

And there's an error hint like "no such directory /root/package.json" however.

**Reply by rarkins** [reply]
It's intentionally an empty string when the package file is in the root directory so errors or hints are not applicable.

**Reply by songkeys** [reply]
Ok. But to your question, there's an error:

```
npm WARN saveError ENOENT: no such file or directory, open '/root/package.json'\nnpm notice created a lockfile as package-lock.json. You should commit this file.\n
```

my commands is like 

```
cd {{packageFileDir}} && npm update ...
```

But the package.json is actually located at the root of the project... What could possibly go wrong here?

**Reply by rarkins** [reply]
Please create a public reproduction to look at.
